silverstripe/framework/SS-2015-006-1.yaml

```diff
@@ -0,0 +1,8 @@
+title:     SS-2015-006: XSS In GridField print
+link:      http://www.silverstripe.org/software/download/security-releases/ss-2015-006/
+cve:       ~
+branches:
+    3.1.x:
+        time:     2015-02-12 15:55:00
```

Comment:
@xabbuh the time here should be the time at which the fix was merged, not the release date (the goal is to be able to know whether dev branches are impacted or no)